src/main/scala/mesosphere/marathon/state/RootGroup.scala

```diff
@@ -203,14 +209,14 @@ class RootGroup(
     *
     * Every transitive group gets the new version.
     *
-    * @param groupId the id of the group to be updated
+    * @param groupId      the id of the group to be updated
```

Comment:
is this some new formatter feature that it tries to align the parameter descriptions? 🤔

Reply:
Maybe intellij did it